### PR TITLE
 v11 tooltip style tab update

### PR DESCRIPTION
**Pull request by kingtraceyj**
Updates on the tooltip style tab for v11

#### Changelog

**Changed**

- updated images and captions
- updated token tables
  - color table: trigger button hover color
  - structure table: spacing for caret on icon button

**Removed**

- definition tooltip images and sections

### CODE REVIEW

File: src/pages/components/tooltip/style.mdx [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-| ------------ | ---------------- | --------------------- |
-| Label        | text color       | `$text-secondary`     |
-| Trigger icon | svg              | `$icon-primary`       |
-| Container    | background-color | `$background-inverse` |
-| Text         | color            | `$text-inverse`       |
+| Element              | Property         | Color token           |
+| -------------------- | ---------------- | --------------------- |
+| Label                | text color       | `$text-secondary`     |
+| Trigger button       | svg              | `$icon-secondary`     |
+| Trigger button hover | svg              | `$icon-primary`       |
```
Comment (aagonzales) [inline src/pages/components/tooltip/style.mdx:18]:
```suggestion
| Trigger button: hover | svg              | `$icon-primary`       |
```

**Review by aagonzales — COMMENTED** [review]
Ok so about the color section images, the style with the lines and text inside the images are actually an old outdated style from v9. There are still a few floating out there that haven't been cleaned up yet. 

Lately, Lauren and I have actually just been using the normal image component (instead of the fixed one) for just the color section since it doesn't have size specs in it. We then divide them up the image like you would for a usage image. Keep any text out of the image and just put that context in the caption. See Breadcrumb, link, checkbox for examples of what i'm talking about.

image

**Review by aagonzales — APPROVED** [review]
Looks great!

**Comment by mjabbink** [comment]
Looks good!

**Review by dakahn — APPROVED** [review]

**Pull request merged**